integration-tests/manager-integration-tests/manager-integration-tests-v1/src/test/java/com/redhat/service/smartevents/integration/tests/v1/RunCucumberTest.java

```diff
@@ -7,12 +7,15 @@
 
 import io.cucumber.core.options.Constants;
 
+import static io.cucumber.junit.platform.engine.Constants.GLUE_PROPERTY_NAME;
 import static io.cucumber.junit.platform.engine.Constants.PLUGIN_PROPERTY_NAME;
 
 @Suite
 @IncludeEngines("cucumber")
 @SelectClasspathResource("features")
 @ConfigurationParameter(key = Constants.FILTER_TAGS_PROPERTY_NAME, value = "not @ansibleaction")
+@ConfigurationParameter(key = GLUE_PROPERTY_NAME,
+        value = "com.redhat.service.smartevents.integration.tests.steps,com.redhat.service.smartevents.integration.tests.resources,com.redhat.service.smartevents.integration.tests.v1.steps")
```

Comment:
Is the `com.redhat.service.smartevents.integration.tests.v1.steps` needed?
That is pointing to package in this module, right. I would expect that packages in module are resolved by Cucumber by default.

Reply:
Done.